packages/app/src/components/nominationShow.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import NominationBanner from './nominationBanner'
+
+const NominationShow = (props) => {
+  const [NominationData, setNominationData] = useState({})
+
+  useEffect(() => {
+    axios.get(`/nomination/${props.match.params.id}`)
```

Comment:
I'm not sure, but we may be trying to push all our actual API calls into the utils/ folder so we can manage them all collectively there (like by setting a base url or authentication).